### PR TITLE
Consistent JSON Output Format

### LINKED ISSUE #94: v-model not working with output-format="json"
Hi,
two-way binding (v-model) not working with output-format="json"
If i re-load json, i see the json, not the result

image

Regards

### PR DESCRIPTION
**Pull request by jlgoff3**
Fixes #94 

### Problem
Currently usage of JSON requires `Raw JSON in -> TipTap v-model -> Serialized JSON out`. The type of the content is object up until the first TipTap edit is made, at which point it becomes a string. My team is using TipTap-Vuetify in our ERP project, and we've previously had to write code to get around this that checks to see whether it is a string or object on save.

### Purpose
This PR is intended to make the behavior of output-format="json" more consistent, namely:
`Raw JSON in -> TipTap v-model -> Raw JSON out.`

**Current behavior** of TipTap-Vuetify with `output-format="json"` is inconsistent. V-model input expects a raw JSON object, but causes [Vue warn] type error and changes output to a serialized JSON object.

**Expected behavior** is that when `output-format="json"`, TipTap-Vuetify accepts a raw JSON object as v-model input with proper type checking and outputs raw JSON object. This PR makes that behavior consistent.

### Replication
[Replication](https://codesandbox.io/s/competent-hooks-7po6k?file=/src/App.vue) - Replication has some issues due to codesandbox's platform limitations. Instructions in the sandbox demonstrate the inconsistent behavior.

Any comments, questions, or recommendations are appreciated. Thank you for your work maintaining this project! It's been very useful in our own project.

### CODE REVIEW

File: src/components/TiptapVuetify.vue [hunk 1]
```diff
@@ -76,285 +67,313 @@ import AbstractExtensionInterface from '~/extensions/AbstractExtensionInterface'
 })
 export default class TiptapVuetify extends Vue {
   @Prop({ type: Boolean, default: false })
-  readonly [PROPS.DISABLED]: boolean
+  readonly [PROPS.DISABLED]: boolean;
 
-  @Prop({ type: String, default: '' })
-  readonly [PROPS.VALUE]: string
+  @Prop({ type: [String, Object], default: "" })
```
Comment (iliyaZelenko) [inline src/components/TiptapVuetify.vue:72]: This is first key moment.

File: src/components/TiptapVuetify.vue [hunk 2]
```diff
@@ -76,285 +67,313 @@ import AbstractExtensionInterface from '~/extensions/AbstractExtensionInterface'
...
+    this.emitAfterOnUpdate = true;
+    let output: any;
 
-    if (this[PROPS.OUTPUT_FORMAT] === 'html') {
-      output = info.getHTML()
+    if (this[PROPS.OUTPUT_FORMAT] === "html") {
+      output = info.getHTML();
     } else {
-      output = JSON.stringify(info.getJSON())
+      output = info.getJSON();
```
Comment (iliyaZelenko) [inline src/components/TiptapVuetify.vue:273]: This is second key moment.

**Comment by iliyaZelenko** [comment]
Hello!

First of all, thanks for a very detailed description of the problem and its solution. It was very informative and I immediately understood the essence of the problem.

It seems like a funny situation arose when your IDE changed the style of the code that does not correspond to my project, which caused ESLint errors on CI.

I see two ways out of the situation:
1) I can take the key points of your changes (I indicated them in the code, maybe I missed something) and make a new Pull Request myself.
2) You will remove unnecessary changes from this PR and I will accept it.

What will we choose?

Thank you for contributing to the project.

**Comment by jlgoff3** [comment]
Oops! I had Prettier enabled on save, and it must have changed the styling. I ran `yarn lint-fix` and pushed, so I believe it should be working now. Thanks for the quick response!

**Comment by iliyaZelenko** [comment]
Now much better, but still removed a lot of line breaks that I would like to see + CSS styles affected.

I suggest copying [the source file](https://github.com/iliyaZelenko/tiptap-vuetify/blob/master/src/components/TiptapVuetify.vue) to return to its original state and add your changes to it. Only a couple of lines will need to be changed again.

**Comment by jlgoff3** [comment]
Hmm, that's what I did last time, but I tried it again, and it seems to have worked. Hope that does it for you. Thanks for working through it with me.

**Comment by jlgoff3** [comment]
Hi, just bumping this to check and see if it's in an acceptable state to be merged. Thanks!

**Comment by iliyaZelenko** [comment]
LGTM, TY!

**Pull request merged**